packages/blade/.storybook/react/preview.tsx

```diff
@@ -180,15 +181,18 @@ export const decorators = [
     return (
       <ErrorBoundary>
         <GlobalStyle />
-        <BladeProvider
-          key={`${context.globals.themeTokenName}-${context.globals.colorScheme}`}
-          themeTokens={getThemeTokens()}
-          colorScheme={context.globals.colorScheme}
-        >
-          <StoryCanvas context={context}>
-            <Story />
-          </StoryCanvas>
-        </BladeProvider>
+        {/* strict in LazyMotion will make sure we don't use excessive `motion` component in blade components and instead use light weight `m` */}
+        <LazyMotion strict features={domMax}>
```

Comment:
Does the consumer need to add the Motion Provider?

If so, this would be a breaking change, correct? Assuming we start using Framer Motion internally for our components as well.

Consider this scenario:

- The consumer is on version 11.37.0.
- In version 11.38.0, we introduce Framer Motion in the BottomSheet component.
- Since the consumer hasn't added the Motion Provider at the root of their app, the BottomSheet component in Blade would stop working.

This means simply running `npm i` to upgrade the Blade version wouldn’t be sufficient in this case.

Reply:
Yup we're making breaking change